### ISSUE TITLE
[v3] lower letters not recognized on password check

### ISSUE THREAD

**Issue by hiroshisuga** [bigbluebutton/greenlight#4756]
**Describe the bug**
The password checker does not allow password. It seems not recognize lower letters. Every alphabet is recognized as capital letters. 

**To Reproduce**
1. Go to Profile - Change password, or Admin - Manage users - +New user
2. Enter password
3. 'At least one lower letter necessary' warning never disappears even when you enter a lower letter
4. If you enter a lower letter, the capital letter warning disappears

**Screenshots**
Here in the demo server, I entered "12345678%aA". The password checker seems to recognise a lower letter as capital.
My browser locale is Japanese, and the locale is set in English here.
<img width="746" alt="スクリーンショット 2023-02-04 8 22 43" src="https://user-images.githubusercontent.com/45039819/216729704-279da9ad-51b7-4d3c-a77a-6077107afaa2.png">

**Comment by farhatahmad** [comment]
Hi @hiroshisuga, is this on the latest version? I can't seem to replicate this 
![image](https://user-images.githubusercontent.com/35435341/216996461-844f3e32-4577-4691-8dad-69b00f559afc.png)

**Comment by farhatahmad** [comment]
Thank you for testing! We really appreciate it. We'll make sure to upgrade demo today

**Comment by hiroshisuga** [comment]
Hi @farhatahmad , I did what you suggested. The password correctly entered. But by this change, the error disappears as well. 
....and finally I found out that the reason is that my iMac uses the Capslock key for switching English/Japanese typing. Usually, I don't have any problem for typing upper and lower cases with the Capslock key on, but only on this form every alphabet that I type seem to be converted into capital letter. 
It's a edge case for Japanese users. But I guess there are more non-western language users who suffer the same problem because this Capslock setting is what Apple recommends.
My suggestion: what about adding an "eye" icon next to the password form. By clicking it, the users can check the password that is converted into plain text (like the solution that you suggested).
